internal/driver/config.go

```diff
@@ -0,0 +1,123 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2023 YIQISOFT Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package driver
+
+import (
+	bootstrapConfig "github.com/edgexfoundry/go-mod-bootstrap/v3/config"
+	"github.com/robinson/gos7"
+)
+
+type ServiceConfig struct {
+	S7Info S7Info
+}
```

Comment:
Not needed unless you plan to have this as custom config in the configuration.yaml file